content/kots-cli/getting-started.md

```diff
@@ -21,11 +21,16 @@ If you're using an [embedded Kubernetes cluster](/kotsadm/installing/installing-
 
 ### Easy installation script
 
-To start, install KOTS using the Krew plugin manager:
+To start, install the latest version of KOTS using the Krew plugin manager:
 ```bash
 curl https://kots.io/install | bash
 ```
 
+You can also install a specific version of KOTS, like one that has been QA'ed by your team, using the versioned install URL:
+```bash
+curl https://kots.io/install/<version> | bash
```

Comment:
Would be nice if there were an example here so you know the format of the version. e.g. with or without the `v` prefix.

Reply:
It works either way, but I can do that.

Reply:
I think originally it had a version in there, but I removed it since we'll never update the reference and people following the getting started guide shouldn't download an ancient version. But I'm sure there's a way I can word it to mitigate that.